### DISCUSSION TITLE
First time hosting on github pages - issues with namecheap custom domain

### DISCUSSION THREAD

**Discussion by jackp5150** [community/community#50169] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi I'm trying to host a custom Namecheap domain on pages and I had the pages working fine with the github domain but am having trouble moving it to the name cheap domain. Any help would be greatly appreciated and I'll post lots of screenshots below.
<img width="1435" alt="Screenshot 2023-03-15 at 3 37 27 PM" src="https://user-images.githubusercontent.com/67347420/225437860-70108ad7-84e4-490b-808e-731a80a07ea2.png">
<img width="1435" alt="Screenshot 2023-03-15 at 3 37 50 PM" src="https://user-images.githubusercontent.com/67347420/225437864-a5376ad6-d18d-4e66-90c0-324bb1267702.png">
<img width="1377" alt="Screenshot 2023-03-15 at 3 38 13 PM" src="https://user-images.githubusercontent.com/67347420/225437866-e91bcff6-a830-4dea-a67a-7945367addbf.png">
<img width="1426" alt="Screenshot 2023-03-15 at 3 38 45 PM" src="https://user-images.githubusercontent.com/67347420/225437871-c7d50279-ea51-4cd8-9842-8b16c2e7ebf8.png">
<img width="1440" alt="Screenshot 2023-03-15 at 3 39 03 PM" src="https://user-images.githubusercontent.com/67347420/225437873-2fd0e4a7-78fa-4b18-a04f-c0bff775d58d.png">
<img width="1440" alt="Screenshot 2023-03-15 at 3 39 10 PM" src="https://user-images.githubusercontent.com/67347420/225437875-25fa4a32-e736-4038-ac55-a109ff25b444.png">
<img width="1394" alt="Screenshot 2023-03-15 at 3 39 22 PM" src="https://user-images.githubusercontent.com/67347420/225437878-05c889c1-ec69-459a-9993-c94e89829f21.png">
<img width="1425" alt="Screenshot 2023-03-15 at 3 39 30 PM" src="https://user-images.githubusercontent.com/67347420/225437880-07cf30ff-822a-4f74-aa4d-9681f2539abf.png">

**Comment by poye-network** [comment]
Hey man, there's nothing wrong, it was moved to your namecheap domain 14mins later after you deployed it. Your site looks great, you did a nice job 👍
